CMSgov/mdct-mcr
# MDCT-1980: Edit Measure Details

## Description

Closes [MDCT-1980](https://qmacbis.atlassian.net/browse/MDCT-1980).

Adding functionality for the "Edit measure" button in **Section C: Access Measures**

### How to test

Run locally!

* Navigate to `/mcpar/program-level-indicators/availability-and-accessibility/access-measures`
* **Add access measure**
* **Edit measure**

Title should say _Edit access measure_, previously-entered data should be hydrating correctly, and when you edit the details, they should be reflecting on the original card.

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
